hack/run-test-serial-e2e.sh

```diff
@@ -196,13 +196,15 @@ function runtests() {
 		${FOCUS}
 }
 
-# Make sure that we always properly clean the environment
```

Comment:
OTOH this should be optional because we still regularly run the serial suite in dev environments (contrary to every other suite)

Reply:
then it's already optional here, because of:
https://github.com/openshift-kni/numaresources-operator/blob/bb5985c336609a958b39258b0594cca7f33f26c4/hack/run-test-serial-e2e.sh#L158
so no changes are needed here